source/clear-linux/tutorials/web-server-install/web-server-install.rst

```diff
@@ -1,479 +1,423 @@
-   sudo systemctl restart httpd.service
+      sudo systemctl enable php-fpm.service
+      sudo systemctl start php-fpm.service
+      sudo systemctl restart httpd.service
 
-After restarting the Apache service, we can test our PHP installation.
+After restarting the Apache service, test your PHP installation.
 
-1. Create a file named :file:`phpinfo.php` in the
-   :file:`/var/www/html/` directory using your editor.
+#. Create a file named :file:`phpinfo.php` in the
+   :file:`/var/www/html/` directory using a text editor.
 
-2. Add the following line to the file:
+#. Add the following line to the file:
 
    .. code-block:: php
 
       <?PHP phpinfo() ?>
 
-3. Go to http://localhost/phpinfo.php using your browser.
+#. Go to http://localhost/phpinfo.php.
 
-   The PHP information screen should appear, see figure 2:
+#. Verify that the PHP information screen appears, as shown in figure 2:
 
    .. figure:: figures/web-server-install-2.png
       :alt: PHP information screen
       :width:     600
 
-      The PHP information screen.
+      `Figure 2: The PHP information screen.`
 
-If the PHP information screen is displayed, you have successfully installed
-the PHP components and are now ready to add your database application to
-complete your LAMP server implementation.
+If the PHP information screen is displayed, you have successfully installed the PHP components and are now ready to add your database application to complete your LAMP server implementation.
 
-Installing MariaDB
+Install MariaDB
 ******************
 
-Most web applications require a database to store their content. Therefore,
-we must install MariaDB to fulfill this need. MariaDB is a drop-in
-replacement for MySQL and is available in the database-basic |CL|
-bundle.
-
-To install the database-basic bundle, enter the following command:
+Install MariaDB to store content. MariaDB is a drop-in replacement for MySQL and is available in the database-basic |CL| bundle.
 
-.. code-block:: console
+#. To install the database-basic bundle, enter the following command:
 
-   sudo swupd bundle-add database-basic
+   .. code-block:: bash
 
-Once MariaDB is installed, we need to start the service and check its status.
+      sudo swupd bundle-add database-basic
 
-To start MariaDB, enter the following commands:
+#. To start MariaDB after it is installed, enter the following commands:
 
-.. code-block:: console
+   .. code-block:: bash
 
-   sudo systemctl enable mariadb
-   sudo systemctl start mariadb
+      sudo systemctl enable mariadb
+      sudo systemctl start mariadb
 
-To check the status of MariaDB, enter the following command:
+#. To check the status of MariaDB, enter the following command:
 
-.. code-block:: console
+   .. code-block:: bash
 
-   sudo systemctl status mariadb
+      sudo systemctl status mariadb
 
-With the MariaDB service running, we can perform some basic security
-hardening.
+With the MariaDB service running, we can perform some basic security hardening.
```

Comment:
In our last meeting we discussed your preference for not using numbered steps in the 'script' section. I recommended that you add a header here to delineate this section.  I recommend that you add a header, as below, to appear above line 219. 
Security Hardening 
---------------------

Reply:
Done. I noticed that the H3 visual did not appear in crtl alt o although H2 did. Just FYI.